vagrant-spk

```diff
@@ -101,6 +99,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     override.vm.synced_folder "..", "/opt/app", type: "9p", accessmode: "passthrough"
     override.vm.synced_folder ENV["HOME"] + "/.sandstorm", "/host-dot-sandstorm", type: "9p", accessmode: "passthrough"
+    override.vm.synced_folder "..", "/vagrant", type: "9p", accessmode: "passthrough"
```

Comment:
I remember now how I avoided using `override`:

I would create a Ruby variable, mutate it depending on the VM provider, and then at the end, use that variable in a call to `config`.

I find the "Use a Ruby variable" way marginally cleaner, but the way you have it is also totally OK.